python/lsst/analysis/ap/apdb.py

```diff
@@ -139,15 +139,15 @@ def load_sources_for_object(self, dia_object_id, exclude_flagged=False, limit=10
             Id of object to load sources for.
         exclude_flagged : `bool`, optional
             Exclude sources that have selected flags set.
-            Use `set_DiaSource_exclude_flags` to configure which flags
+            Use `set_excluded_diaSource_flags` to configure which flags
             are excluded.
         limit : `int`
             Maximum number of rows to return.
 
         Returns
         -------
         data : `pandas.DataFrame`
-            DiaSources for the specified diaObject.
+            diaSources for the specified diaObject.
```

Comment:
So I guess I left out a case in the capitalization wars, and I've asked for clarification on slack about what to do when there's a lowercase thing at the start of a sentence.

Reply:
Given the slack thread, I think here we'd either keep `DiaSources` capitalized at the start of sentences, or put it in double-backticks (which for consistency we'd want to do everywhere). I think I'd lean towards just capitalizing things at the start of sentences, to avoid excessive back-ticking?

Reply:
I took the ambiguous/easy way out and rephrased wherever lowercase things were starting sentences.